redis/src/commands/macros.rs

```diff
@@ -54,7 +54,16 @@ macro_rules! implement_commands {
                 c.iter(self)
             }
 
+            /// Incrementally iterate the keys space with options.
+            #[inline]
+            fn scan_options<RV: FromRedisValue>(&mut self, opts: ScanOptions) -> RedisResult<Iter<'_, RV>> {
```

Comment:
Hi, thanks for the PR! 

I think the cursor parameter should probably not be part of options, since it's required. In other commands with the same pattern, the options struct is for optional arguments.

Reply:
hmm, ok, I can hardcode it to 0, I am not that profound redis user and I just thought that there are use cases when one would want to iterate using the cursor returned from redis

Reply:
The `Cmd::iter/_async` functions return an iterator that updates the cursor internally, so there's no need to set it externally.